Makefile

```diff
@@ -271,6 +268,8 @@ tools/bin/misspell:tools/check/go.mod
 tools/bin/ineffassign:tools/check/go.mod
 	cd tools/check; \
 	$(GO) build -o ../bin/ineffassign github.com/gordonklaus/ineffassign
+tools/bin/golangci-lint:
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b ./tools/bin v1.21.0
```

Comment:
Will the binary from this URL change? Is there any version control for tools?
If the `make` result depends on the tool's version, it may trouble us sometimes. @xiekeyi98 

We use go.mod to control the version of other tools here: 
https://github.com/pingcap/tidb/blob/master/tools/check/go.mod

Reply:
Well, version control for tool is at last ( ` v1.21.0` ).
What do you think about we should do to install it.? I use this tool: https://github.com/golangci/golangci-lint